index.html

```diff
@@ -4,13 +4,234 @@
+              </li>
+              <li class="main-program-ul-li topic">Lecture</li>
+              <li class="main-program-ul-li">
+                listen to the speaker from various country about sharing and
+                opening
+              </li>
+            </ul>
+          </div>
+          <button class="join-button text-align-center">
+            Join CC Summit 2015
+          </button>
+        </div>
+
+        <div class="featured-speaker-wrapper">
+          <h2 class="text-align-center">Food Expert speaker</h2>
+          <ul
+            class="featured-speakers-ul d-flex j-content-btw f-direction-col flex-d-row-lg f-wrap align-items"
+          ></ul>
+        </div>
+      </section>
+    </main>
+    <footer class="footer">
+      <h3 class="text-align-center">Partner</h3>
+      <ul
+        class="d-flex j-content-cnt align-items f-direction-col flex-d-row-lg footer-ul"
+      >
+        <li class="d-flex align-items j-content-btw flex-d-collg">
+          <img
+            class="footer-image"
+            src="./images/united-nation-logo.png"
+            alt="united nation logo"
+          />
+          <p><a class="footer-image-title">UNITED NATIONS</a></p>
+        </li>
+
+        <li class="footer-2-ul-li">
+          <ul class="d-flex f-direction-row j-content-cnt align-items f-wrap">
+            <li class="social-media-li">
+              <a
+                ><img
+                  src="./images/github-icon.png"
+                  class="social-media-image"
+                  alt="github icon"
+              /></a>
+            </li>
+
+            <li class="social-media-li">
+              <a
+                ><img
+                  src="./images/linkdin.svg"
+                  class="social-media-image"
+                  alt="linkdin icon"
+              /></a>
+            </li>
+            <li class="social-media-li">
+              <a
+                ><img
+                  src="./images/twitter.png"
+                  class="social-media-image"
+                  alt="twitter icon"
+              /></a>
+            </li>
+
+            <li class="social-media-li">
+              <a><p class="mozilla">mozilla</p> </a>
+            </li>
+
+            <li class="social-media-li">
+              <a
+                ><img
+                  src="./images/angels-list.png"
+                  class="social-media-image"
+                  alt="angels list icon"
+              /></a>
+            </li>
+
+            <li class="social-media-li">
+              <a
+                ><img
+                  src="./images/medium.png"
+                  class="social-media-image"
+                  alt="medium logo"
+              /></a>
+            </li>
+
+            <li class="daumakakau">
+              <a><p>daumakakau</p> </a>
+            </li>
+          </ul>
+        </li>
+      </ul>
+    </footer>
+    <script src="mobile-menu.js" defer></script>
```

Comment:
- [ ] - [ ] Please follow the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and implement the required changes on your partners and footer sections on both desktop and mobile version
![5](https://user-images.githubusercontent.com/47625407/157782409-f33ced2a-a529-4ec7-b1d1-f0406d3459f8.png)

Reply:
Thanks for the review change made